app/models/user.rb

```diff
@@ -1510,4 +1515,19 @@ def anonymize(new_wca_id = nil)
       country_iso2: special_account? ? country_iso2 : nil,
     )
   end
+
+  def transfer_data_to(user)
```

Comment:
In the interest of readability, the parameter should probably be called `new_user` or `target_user` or similar.